client/src/components/ContentNavigator/ContentModel.ts

```diff
@@ -113,9 +113,7 @@ export class ContentModel {
     } else if (filters.length > 1) {
       membersUrl = membersUrl + "&filter=and(" + filters.join(",") + ")";
     }
-    membersUrl =
-      membersUrl +
-      "&sortBy=eq(contentType,'folder'):descending,name:primary:ascending,type:ascending";
+    membersUrl = membersUrl + "&sortBy=type:descending,name:primary:ascending";
```

Comment:
membersUrl =
      membersUrl +
      `&sortBy=${
        parentIsContent ? "" : "eq(contentType,'folder'):descending,"
      }name:primary:ascending,type:ascending`;

Reply:
That seemed to work :)